Article/Article.js

```diff
@@ -3,27 +3,38 @@
 class Article {
   constructor(domElement) {
```

Comment:
`domElement` is a banana name, you could have also use `atricle`  `this.article=article` as long as you are consistant